.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[ci skip]')"
     strategy:
       matrix:
-        node-version: [14.16.1, 14.17]
+        node-version: [14.17]
```

Comment:
Dropping the oldest version.
I didn't update to a newer one because currently it breaks the test pipeline, since dependency hell in https://github.com/adobe/asset-compute-sdk/pull/213.